RunCat365/Program.cs

```diff
@@ -146,6 +146,19 @@ public RunCat365ApplicationContext()
             cpuTimer.Start();
         }
 
+        private static Image? GetIconImageForMenu(string title)
+        {
+            Theme systemTheme = GetSystemTheme();
+            string prefix = systemTheme.GetString();
+            string iconName = $"{prefix}_{title}_0".ToLower();
+            var obj = Resources.ResourceManager.GetObject(iconName);
+            return obj switch
+            {
+                Icon icon => icon.ToBitmap(),
+                _ => null
+            } ?? null;
+        }
```

Comment:
Icons are not necessary for anything other than runners, so let's specialize them.

```suggestion
        private static Bitmap? GetRunnerThumbnailBitmap(Runner runner)
        {
            var systemTheme = GetSystemTheme();
            var iconName = $"{systemTheme.GetString()}_{runner.GetString()}_0".ToLower();
            var obj = Resources.ResourceManager.GetObject(iconName);
            return obj is Icon icon ? icon.ToBitmap() : null;
        }
```